src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -55,6 +91,12 @@ export const ConfigurePatchCore: React.FC<Props> = ({ patch }) => {
   }
   return (
     <>
+      {errorSchedulingPatch && (
+        // TODO: replace with error banner
+        <div data-cy="error-banner">
+          There was a problem trying to schedule patch.
+        </div>
+      )}
```

Comment:
Do you see errors being displayed in the ui on the component level where they occurred or as more of a global error at the page level. As a banner that appears across the top of the screen. or as toast notification

Reply:
i like toast notifications but the designers decided on banners, at least for now

Reply:
I think banners are used throughout mongo so they are probably trying to keep a similar design language. University and stitch both use them.